tasklist/client/src/index.scss

```diff
@@ -70,3 +70,7 @@ body,
 svg {
   fill: currentcolor;
 }
+
+.cds--overflow-menu-options {
+  width: unset;
+}
```

Comment:
@volodymyr-melnykc Why is this necessary?

Reply:
Right this is a fix to allow the sorting dropdown to expand. The default carbon component is super limited in width and they just don't support it being wider.

Reply:
@vsgoulart The width of the popover was fixed which caused the issue that the text in other languages wasn't fully displayed (truncated).

![image](https://github.com/user-attachments/assets/75437ef2-2380-4031-838b-798af2a67971)

Reply:
With this, it adapts to the width of its contents. I wonder if we want to improve upon this, but I think solving it the proper way (which is to ask carbon for a wider one lol) would probably take too long. We can follow up IMO.

Reply:
@Skaiir  @volodymyr-melnykc We still had some ellipsis in Spanish for me. I fixed it now and made the style overwrite localized to that component only instead of globally like it was before.

Please check my last commit